Cargo.toml

```diff
@@ -36,11 +36,17 @@ tempfile = "3.8"
 arrow2 = "0.18"
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
```

Comment:
is it necessary to patch it here or in `gdal-src`?

Reply:
For cargo workspaces that section needs to be in the workspace `Cargo.toml`:

> Cargo only looks at the patch settings in the Cargo.toml manifest at the root of the workspace. Patch settings defined in dependencies will be ignored.

[Source](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section) (last sentence)